ion-schema-tests-runner/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ion-schema-tests-runner"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
FYI—`ion-schema-tests-runner` is not published to crates.io, nor does any crate depend on it. There's no need to change this version.